### PR TITLE
Python3 support

### PR DESCRIPTION
**Pull request by manuelgavidia**

### CODE REVIEW

File: alarmlibrary/alarm.py [hunk 1]
```diff
@@ -13,13 +13,13 @@ class AlarmSeverity(Enum):
 
 class Alarm(object):
 
-    def __init__(self, domain, namespace, severity, timestamp=long(time.time()), description=""):
+    def __init__(self, domain, namespace, severity, timestamp=int(time.time()), description=""):
```
Comment (mmagr) [inline alarmlibrary/alarm.py]: On a wider note, do we really need to cast to int?
Do notice that we may very well (for larger deploys) have way more than a given alarm instance per second.

Comment (manuelgavidia) [reply alarmlibrary/alarm.py]: Yep, this is from legacy code. I will change it

Comment (manuelgavidia) [reply alarmlibrary/alarm.py]: But, alarm manager can not handle float time data. It expects an integer. So?

Comment (mmagr) [reply alarmlibrary/alarm.py]: (alarmManager: :disappointed:: )

keep the int then...

**Pull request merged**